src/app.scss

```diff
@@ -0,0 +1,4 @@
+@import './components/TutorialHeader/tutorial-header.scss';
+@import './components/TutorialHeader/tutorial-header.scss';
```

Comment:
You have a duplicate import here.

Reply:
This is still not resolved and your Netlify preview build is failing: https://app.netlify.com/sites/carbon-tutorial/deploys/5d4c044d57bd6b0007a9cbb7

Reply:
My view says something different
[https://github.com/carbon-design-system/carbon-tutorial/pull/2350/files#diff-21e3718002c800ff2f9682b9f454a67f]

Reply:
the preview build is from 8 August, do you have a current one?

Reply:
https://app.netlify.com/sites/carbon-tutorial/deploys/5d513797c1a9fe000862c5fc